process/smartContract/hooks/blockChainHook.go

```diff
@@ -37,6 +37,11 @@ var log = logger.GetOrCreate("process/smartcontract/blockchainhook")
 const defaultCompiledSCPath = "compiledSCStorage"
 const executeDurationAlarmThreshold = time.Duration(50) * time.Millisecond
 
+// TODO move these in vmcommon, make a new type for them and make them exported
```

Comment:
Better convert into an enumeration, actually.

Reply:
converted to error, actually, looks cleaner